app/services/postcard.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var Config = require('../../config');
+
+module.exports = function (API) {
+
+  var Postcard = {};
```

Comment:
I'm not sure if `postcard-create` has it yet, but our `dashboard` is set up to be able to handle the ES6 `let`s, `const`s, and arrow functions. 

No need to change here if it's not supported. I believe we had to add something special.